x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -733,6 +733,7 @@ observe the activity of the input.
 | `sqs_messages_returned_total`             | Number of SQS message returned to queue (happens on errors implicitly after visibility timeout passes).
 | `sqs_messages_deleted_total`              | Number of SQS messages deleted.
 | `sqs_message_processing_time`             | Histogram of the elapsed SQS processing times in nanoseconds (time of receipt to time of delete/return).
+| `sqs_message_delayed_time`                | Histogram of delay in SQS processing times in nanoseconds (message sent time to now).
```

Comment:
How about:

Histogram of the difference between the SQS `SentTimestamp` attribute and the time when the SQS message was received expressed in nanoseconds.